src/cmd/build.rs

```diff
@@ -14,6 +15,17 @@ pub fn build(
 ) -> Result<()> {
     let mut site = Site::new(root_dir, config_file)?;
     if let Some(output_dir) = output_dir {
+        // Check whether output directory exists or not
+        // This way we don't replace already existing files.
+        if output_dir.exists() {
+            console::warn("The directory you are about build to already exists. Building to this directory will delete files contained within this directory.");
+
+            // Prompt the user to ask whether they want to continue.
+            if !ask_bool("Are you sure you want to continue?", false)? {
```

Comment:
This makes `zola build` potentially interactive with no way of skipping it. Either we add a flag to skip prompts or we add a timeout to the prompts

Reply:
As I mentioned in the PR:

>There is one concern I ran into after implementing this. It may need to be discussed still. Is it okay to require CI's to first delete the folder in question? This may break CI's and automated scripts, and therefore it may be interesting to add to the 0.15 release.

A timeout would seem to be okay for now, but this would still change default behavior if the default value is `false`. I feel like `false` as default value is the correct option, though.

Reply:
Yep I've seen that comment but I think there needs a way to just ignore prompts. I lean towards timeout